CHANGES/.gitignore

```diff
@@ -0,0 +1,2 @@
+!.gitignorie
```

Comment:
I guess this is to add empty directory to `git`?
I've seen another pattern, adding `.gitkeep` file to git.

Reply:
yes it's to retain the directory when empty. yeah `.gitkeep` seems better